build_alpha.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-./gradlew clean build
-cp app/build/outputs/apk/app-alpha-matrixorg.apk ./alpha.apk
```

Comment:
Because this is the SDK  project, and it can't produce an apk
On Feb 17, 2016 3:43 PM, "Matthew Hodgson" <notifications@github.com> wrote:

> In build_alpha.sh
> <https://github.com/matrix-org/matrix-android-sdk/pull/36#discussion_r53172575>
> :
>
> > @@ -1,3 +0,0 @@
> > -#!/bin/bash
> > -./gradlew clean build
> > -cp app/build/outputs/apk/app-alpha-matrixorg.apk ./alpha.apk
>
> Why is build_alpha.sh being removed here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matrix-org/matrix-android-sdk/pull/36/files#r53172575>
> .
>